botstrap.py

```diff
@@ -40,14 +46,71 @@ def __init__(self):
         super().__init__(
             base_url="https://discord.com/api/v10",
             headers={"Authorization": f"Bot {BOT_TOKEN}"},
-            event_hooks={"response": [self._raise_for_status]}
+            event_hooks={"response": [self._raise_for_status]},
         )
 
     @staticmethod
     def _raise_for_status(response: Response) -> None:
         response.raise_for_status()
 
 
+def upgrade_server_to_community_if_necessary(
+        guild_id: int | str,
+        rules_channel_id_: int | str,
+        announcements_channel_id_: int | str,
+        client: DiscordClient) -> None:
+    """Fetches server info & upgrades to COMMUNITY if necessary."""
+    response = client.get(f"/guilds/{guild_id}")
+    payload = response.json()
+
+    if COMMUNITY_FEATURE not in payload["features"]:
+        log.warning("This server is currently not a community, upgrading.")
+        payload["features"].append(COMMUNITY_FEATURE)
+        payload["rules_channel_id"] = rules_channel_id_
+        payload["public_updates_channel_id"] = announcements_channel_id_
+        client.patch(f"/guilds/{guild_id}", json=payload)
+
+
+def upgrade_server_to_community(guild_id: int | str, client: DiscordClient) -> None:
+    """
+    Transforms a server into a community one.
+
+    Return true if the server has been correctly upgraded, False otherwise.
```

Comment:
This docstring doesn't match the function. What is the purpose of this function? It's not used anywhere in this PR.

Reply:
Oh... No it's an old version that just escaped my eyes.
It needs to go away indeed

Reply:
00330956f6140476d9e9f633b045921b214b3d37

Reply:
Haha, I thought so. I'll approve after i get a chance to test the code.